komga-webui/src/locales/en.json

```diff
@@ -427,22 +427,22 @@
     },
     "filename_chooser": {
       "button_choose": "Choose",
-      "field_destination_filename": "Destination file name",
-      "label_source_filename": "Source File Name",
+      "field_destination_filename": "Destination filename",
+      "label_source_filename": "Source filename",
       "table": {
         "existing_file": "Existing File",
         "order": "Order"
       },
-      "title": "Destination File Name"
+      "title": "Destination Filename"
     },
     "password_change": {
       "button_cancel": "Cancel",
       "button_confirm": "Change password",
       "dialog_title": "Change password",
       "field_new_password": "New password",
-      "field_new_password_error": "New password is required.",
-      "field_repeat_password": "Repeat new password",
```

Comment:
Shorter. Contention.

Reply:
what do you mean when you say "Contention" ?

Reply:
You could argue otherwise here.